Formula/ghc.rb

```diff
@@ -15,6 +15,10 @@ class Ghc < Formula
   deprecated_option "tests" => "with-test"
   deprecated_option "with-tests" => "with-test"
 
+  # https://www.haskell.org/ghc/download_ghc_8_0_1#macosx_x86_64
+  # "This is a distribution for Mac OS X, 10.7 or later."
```

Comment:
This comment should live above the `resource` I think as that's the thing that we need to check the supported versions on.